### ISSUE TITLE
[F-VLM] precomputed embeddings for lvis 1.0?

### ISSUE THREAD

**Issue by earl-juanico** [google-research/google-research#1730]
That worked, thanks!

_Originally posted by @aakb101 in https://github.com/google-research/google-research/issues/1729#issuecomment-1706306836_
            
@weichengkuo, thank you. I also confirm that this was the command modification needed to run the demo code. I am a grad student and very interested in confirming the results reported. I was able to confirm the evaluation on COCO using the precomputed embeddings provided. I also wish to confirm the evaluation on LVIS. Would it be possible for you to upload the precomputed embeddings for LVIS 1.0?

**Comment by weichengkuo** [comment]
Hi @earl-juanico, thanks for verifying the evaluation on COCO using the provided embeddings. I have prepared the LVIS precomputed embeddings at this URL: [https://storage.googleapis.com/cloud-tpu-checkpoints/detection/projects/fvlm/lvis_embeddings.zip](https://storage.googleapis.com/cloud-tpu-checkpoints/detection/projects/fvlm/lvis_embeddings.zip). You can download it with wget or other tools for LVIS evaluation. Hope that helps!

**Comment by earl-juanico** [comment]
Thank you so much @weichengkuo . I will test the embeddings soon. :)

**Comment by earl-juanico** [comment]
I have verified the evaluation on LVIS 1.0 with the embeddings. Thank you @weichengkuo. Any special reason why the precomputed category embeddings have for both COCO and LVIS datasets have 1601 rows?

**Comment by weichengkuo** [comment]
Hi @earl-juanico, they are padded to 1601 due to the background (first entry) and empty embeddings (remaining entries). FYI - https://github.com/google-research/google-research/issues/1731. Thanks!

**Comment by DMUgjj** [comment]
@earl-juanico Hi, have you successfully reproduced the experimental results from the paper? Including COCO or LVIS?

**Comment by DMUgjj** [comment]
@earl-juanico Hi, have you successfully reproduced the experimental results from the paper? Including COCO or LVIS?

**Comment by earl-juanico** [comment]
Hi @DMUgjj. I have so far been able to do it for the RN50 backbone. Next up, will be RN50x4 and RN50x16. 

Hello @weichengkuo, thank you so much for providing the precomputed embeddings. I am still curious if you have tried other padding sizes, e.g., less than 1601 or more than 1601, like a hyperparameter perhaps?

**Comment by DMUgjj** [comment]
@earl-juanico Hi, how did you reproduce the results of their paper? Using the author's code? I tried to reproduce the results of R50 in the paper on coco dataset but did not reproduce its results. I hope you give detailed steps to reproduce it. Did you make your own training set containing only the base categories to train the model?
Thanks for your help and reply!

**Comment by chenxi52** [comment]
i found the emb[:83] is different from each other, but should be 1 background and 80 classes?
